.circleci/config.yml

```diff
@@ -127,7 +127,7 @@ jobs:
               --labels=backport
       - deploy:
           name: Publish plugin to Jetbrains Marketplace
-          command: ./gradlew publishPlugin
+          command: ./gradlew --max-workers=2 publishPlugin
```

Comment:
That's a workaround - the project still gets unnecessarily recompiled to certain extent. Probably to actually make sure that it doesn't happen I'd need to fix our use GrammarKit plugin, but that's not as urgent.